ddtrace/llmobs/_llmobs.py

```diff
@@ -670,8 +678,14 @@ def _start_span(
         session_id = session_id if session_id is not None else _get_session_id(span)
         if session_id is not None:
             span._set_ctx_item(SESSION_ID, session_id)
+
+        ml_app = ml_app if ml_app is not None else _get_ml_app(span)
```

Comment:
nit: the `ml_app if ml_app is not None` is duplicated here and in `_get_ml_app`

Reply:
actually this one might be needed as the check in `_get_ml_app` is

```python
ml_app = span._get_ctx_item(ML_APP)
if ml_app:
    return ml_app
```

which atp the ml_app has not been set on the span yet, so we need to check in the caller. but correct me if i'm wrong!